week1/conf/index-bbuy-queries.logstash.conf

```diff
@@ -0,0 +1,45 @@
+input {
+  file {
+      #Put in the path to the train CSV query data.  To start smaller, simply select a smaller number of files or copy a few files to a different directory.
+      path => ["/workspace/datasets/train.csv"]
+      mode => "read"
+      exit_after_read => true
+      sincedb_path => "/dev/null"
+      file_completed_action => "log"
+      file_completed_log_path => "/dev/null"
+  }
+}
+
+filter {
+  
+  csv {
+      autodetect_column_names => true
+      skip_header => true
+  }
+
+  date {
+      match => [ "click_time", "yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS" ]
+      target => "click_time"
+  }
+
+  date {
+      match => [ "query_time", "yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS" ]
+      target => "query_time"
+  }
+  
+  mutate {
+     # clean up some stuff we don't need
```

Comment:
I was able to add some convert to mutation and change the types from string to float
e.g.
        convert => {
          "regularPrice" => "float"
        }